### PR TITLE
Cslotboom updates

### PR DESCRIPTION
**Pull request by cslotboom**
I'm a little confused about the comparison with the main branch on this pull request - some of these changes were accepted in #20, but some old files are still showing up as new? Maybe I'm missing something.

Anyways, what should be in this update is the following:
- Changed the internal functions to return matplotlib objects

In the get_rendering_animation() file:

- added additional internal functions create the plot objects for a 2D or 3D model 
- Added a function that initializes the figure
- Added a sample function that plots the model using the prior functions. This is only included as a sample for our internal reference for now.

### CODE REVIEW

File: openseespy-pip/openseespy/postprocessing/internal_plotting_functions.py [hunk 1]
```diff
@@ -21,118 +21,154 @@
...
-			surfZarray = np.array([[aNode[2], dNode[2]], [bNode[2], cNode[2]]])
-				
-			if fillSurface == 'yes':
-				ax.plot_surface(surfXarray, surfYarray, surfZarray, edgecolor='k', color='g', alpha=.5)
-			
-			del aNode, bNode, cNode, dNode, surfXarray, surfYarray, surfZarray
-
-
-def _plotCubeVol(iNode, jNode, kNode, lNode, iiNode, jjNode, kkNode, llNode, ax, show_element_tags, element, eleStyle, fillSurface):
+def _plotCubeSurf(nodeCords,ax,fillSurface,eleStyle):
```
Comment (u-anurag) [inline openseespy-pip/openseespy/postprocessing/internal_plotting_functions.py:24]: Where did the eleStyle input go?

**Comment by u-anurag** [comment]
I guess you should update your fork to match with this branch before a pull request to avoid conflicts. These conflicts should not be there. Also, here are some things I observed,

1) Do we need a separate file (`Get_Rendering_Animation`) for animation functions? I think it is possible to edit the existing functions/file to return objects for animations. With extra file/functions, it will be difficult to streamline the changes (if any) in the future.
2) In internal_plotting_functions, all the internal plotting definitions (except _plotCubeSurf and _plotCubeVol) return an object `tempLines` but it is not used anywhere. Is there a disconnect somewhere?
3) The `eleStyle` input is missing from _plotCubeSurf() internal function. Is there a reason for that. Looks like it was edited in the previous PR and I missed it.
3) The animation library uses an internal function called _plotEle_2D and _plotEle_3D that use the existing internal plotting functions. They return objects as figLines, figSurfaces, figTags to help with the animation. If we have objects returned here, do we still need objects returned in the existing internal plotting functions?
4) I do not see any animation function in `Get_Rendering_Animation` at the end. It only has `sample_plot_model()` and returned objects. Is that file complete?

**Comment by cslotboom** [comment]
You are right - I'm going to merge the branches and try again. Sorry about the confusion. This will likely address some of the existing issues.

1. For now I had the functions in a separate file - this is intended to be temporary. It does make sense to clean it up and merge them eventually, however I didn't want to make any big changes in the main file. My thought was once we agree on some of the functionality, we can merge the functions.

2. Let me check this and confirm.
3. Let me check this and confirm.
4. We still need to return the objects, because _plotEle_2D and _plotEle_3D use the internal functions.
5. The animation will follow shortly

**Comment by cslotboom** [comment]
I'm going to delete this pull and use one that removes conflicts.

**Pull request closed without merging**